net/pfSense-pkg-pfBlockerNG-devel/pkg-plist

```diff
@@ -35,6 +35,9 @@ www/widgets/javascript/pfblockerng.js
 www/widgets/widgets/pfblockerng.widget.php
 www/wizards/pfblockerng_wizard.xml
 www/wizards/pfblockerng_wizard.inc
+var/unbound/pfb_unbound.py
+var/unbound/pfb_unbound_include.inc
+var/unbound/pfb_py_hsts.txt
 %%DATADIR%%/info.xml
```

Comment:
These 3 lines must start with `/`, otherwise it considers them to be installed under `${PREFIX}`.  It will require to add `@dir /var/unbound` after them so the directory itself it removed if empty on pkg uninstall

Reply:
ok